arch/arm/boot/dts/overlays/revpi-flat-s-2022-overlay.dts

```diff
@@ -0,0 +1,368 @@
+/*
+ * Device tree overlay for Revolution Pi by KUNBUS
+ *
+ * RevPi Flat S (2022)
+ */
+
+/dts-v1/;
+/plugin/;
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/interrupt-controller/irq.h>
+#include <dt-bindings/pinctrl/bcm2835.h>
+
+/{
+	compatible = "brcm,bcm2711";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			compatible = "kunbus,revpi-flat-s-2022",
+				     "kunbus,revpi-flat", "brcm,bcm2711";
+
+			aout_vref: fixedregulator_2v5x4 {
+				compatible = "regulator-fixed";
+				regulator-name = "2v5_x4_amplified";
+				regulator-min-microvolt = <10000000>;
+				regulator-max-microvolt = <10000000>;
+				regulator-always-on;
+			};
+
+			ain_vref: fixedregulator_2v5x5 {
+				compatible = "regulator-fixed";
+				regulator-name = "2v5_x5_amplified";
+				regulator-min-microvolt = <12500000>;
+				regulator-max-microvolt = <12500000>;
+				regulator-always-on;
+			};
+
+			wl_on: reg_wl {
+				compatible = "mmc-pwrseq-simple";
+				reset-gpios = <&expander 10 GPIO_ACTIVE_LOW>;
+			};
+
+			leds {
+				compatible = "gpio-leds";
+				power_red {
+					gpios = <&gpio 24 GPIO_ACTIVE_HIGH>;
+					linux,default-trigger = "power_red";
+				};
+				a1_green {
+					gpios = <&expander 0 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a1_green";
+				};
+				a1_red {
+					gpios = <&expander 1 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a1_red";
+				};
+				a2_green {
+					gpios = <&expander 2 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a2_green";
+				};
+				a2_red {
+					gpios = <&expander 3 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a2_red";
+				};
+				a3_green {
+					gpios = <&expander 4 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a3_green";
+				};
+				a3_red {
+					gpios = <&expander 5 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a3_red";
+				};
+				a4_green {
+					gpios = <&expander 6 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a4_green";
+				};
+				a4_red {
+					gpios = <&expander 7 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a4_red";
+				};
+				a5_green {
+					gpios = <&expander 8 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a5_green";
+				};
+				a5_red {
+					gpios = <&expander 9 GPIO_ACTIVE_LOW>;
+					linux,default-trigger = "a5_red";
+				};
+				led-act {
+					status = "disabled";
+				};
+			};
+
+			gpio-keys {
+				compatible = "gpio-keys";
+				pinctrl-names = "default";
+				pinctrl-0 = <&button_pins>;
+				user-event {
+					label = "User event key";
+					gpios = <&gpio 23 GPIO_ACTIVE_LOW>;
+					linux,code = <KEY_UNKNOWN>;
+				};
+			};
+
+			sd8xxx-wlan {
+				cfg80211_wext = <0xf>;
+				cal_data_cfg = "none";
+				max_vir_bss=<1>;
+				ps_mode=<1>;
+				auto_ds=<1>;
+				host_mlme=<1>;
+				fw_name="nxp/sdiouartiw416_combo_v0.bin";
+                        };
```

Comment:
Indentation error.

Rest looks fine to me. I don't know what else will be needed, but with this basic set of configuration parameters the wifi test was successful. We can check with ublox or Spezial Electronics if other parameters are usefull, if necessary or if we see any weird behaviour.

Reply:
![image](https://user-images.githubusercontent.com/102968928/189677773-3b12d07c-81f9-4557-bf37-c64c12544cfb.png)

Tabs vs. Spaces ;)

Reply:
Indeed, I will correct this, thx!